src/DustyBot/Modules/BotModule.cs

```diff
@@ -351,6 +354,64 @@ public async Task RemoveSupporter(ICommand command)
             await command.ReplySuccess(Communicator, $"Removed {removed} supporters.");
         }
 
+        [Command("번역", "Translate words and sentences.")]
+        [Alias("tr")]
+        [Parameter("start", ParameterType.String, "Select the language that needs to be translated.")]
+        [Parameter("end", ParameterType.String, "Select the language to translate.")]
+        [Parameter("Message", ParameterType.String, ParameterFlags.Remainder, "Input the word or sentence you want to translate.")]
+        public async Task Translation(ICommand command)
+        {
+            var config = await Settings.ReadGlobal<BotConfig>();
+
+            await command.Message.Channel.TriggerTypingAsync();
+            string stringMessage = command["Message"].ToString();
+            string FirstLang = command["start"].ToString();
+            string LastLang = command["end"].ToString();
+
+            var data = new Dictionary<string, string>()
+            {
+                {"source", FirstLang},
+                {"target", LastLang},
+                {"text", stringMessage},
+            };
+
+            //TODO(BJRambo) : checking to why did Reload httpclient later.
+            using (var papagoClient = new HttpClient())
```

Comment:
This is a wrong usage of HttpClient, check https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/
Use `WebRequest.CreateHttp` for now please. I plan to switch to IHttpClientFactory in all commands later.